### DISCUSSION TITLE
ZIP code sometimes lowercase&no spaces

### DISCUSSION THREAD

**Discussion by bindalf** [etsy/open-api#1218] [Q&A]
in some orders the ZIP Code in the getOrder json response is different that the info on the website
For example the right zip code from Canada is "H7P 5T7" but the api returns "h7p5t7" which causes problems when creating a shipping label.

![zip](https://github.com/etsy/open-api/assets/123418968/923276a1-c15b-4efc-ae96-09a5f470e4df)

**Comment by Gareth064** [comment]
It's possible that the user entered it in lower case, so that's what's stored in Etsy. And on the Etsy UI they are performing a ToUpper() on all post/zip codes.

**Reply by bindalf** [reply]
the actual problem is the missing space
